examples/layers/analysis_options.yaml

```diff
@@ -0,0 +1,13 @@
+# Use the parent analysis options settings and enable null-experiment.
```

Comment:
Another option is to remove the layers example from the `flutter` framework tests and run them separtely